application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -55,25 +58,27 @@ public Schedule createSchedule() {
     public void runRoutine(JDA jda) {
         Instant now = Instant.now();
         database.write(context -> context.selectFrom(PENDING_REMINDERS)
-            .where(PENDING_REMINDERS.REMIND_AT.lessOrEqual(now))
-            .stream()
-            .forEach(pendingReminder -> {
-                sendReminder(jda, pendingReminder.getId(), pendingReminder.getChannelId(),
-                        pendingReminder.getAuthorId(), pendingReminder.getContent(),
-                        pendingReminder.getCreatedAt());
-
-                pendingReminder.delete();
-            }));
+                .where(PENDING_REMINDERS.REMIND_AT.lessOrEqual(now))
+                .stream()
+                .forEach(pendingReminder -> {
+                    sendReminder(jda, pendingReminder.getId(), pendingReminder.getChannelId(),
+                            pendingReminder.getAuthorId(), pendingReminder.getContent(),
+                            pendingReminder.getCreatedAt(), pendingReminder.getGuildId());
+
+                    pendingReminder.delete();
+                }));
     }
 
-    private static void sendReminder(JDA jda, long id, long channelId, long authorId,
-            CharSequence content, TemporalAccessor createdAt) {
+    private void sendReminder(JDA jda, int id, long channelId, long authorId,
```

Comment:
why did u change the id to `int`? isnt it a `long`? what does `pendingReminder.getId()` return?

Reply:
![image](https://user-images.githubusercontent.com/61616007/220454155-93fd6c1d-abc6-4096-b2f8-eee42082e8b8.png)
The table's ID field was Integer, so i think that's why i changed it.

Reply:
ah. please note that sqlites `INTEGER` type is also for `long`. its a variable size integer :)

Reply:
so im not sure whether to keep it as `int` type or switch back to `long`?

Reply:
`long` please :)

Reply:
ah. okay. if the autogenerated code has `int`, then im okay with having `int`.